sycl/test-e2e/ESIMD/lsc/atomic_smoke.cpp

```diff
@@ -779,7 +778,6 @@ bool test_int_types_and_sizes(queue q, const Config &cfg) {
   return passed;
 }
 
-#ifndef USE_DWORD_ATOMICS
```

Comment:
Why did you remove this macro-guard?
Don't these tests verify fadd/fmin available only on LSC. The called API is old, but it is automatically converted (with a warning) to LSC. Is that correct?

Reply:
I see now, the FAdd is still guarded, so, this fix is good